rkt/gc.go

```diff
@@ -179,6 +179,49 @@ func emptyGarbage() error {
 	return nil
 }
 
+// cleanExitedGarbage prepares the pod's directory for removal. If anything
+// fails here then we can conclude that the pod's directory is in a bad state
+// and needs to be cleaned up forcibly.
+func cleanExitedGarbage(p *pod, s *store.Store) error {
+	stage1TreeStoreID, err := p.getStage1TreeStoreID()
+	if err != nil {
+		return fmt.Errorf("error getting stage1 treeStoreID: %v", err)
+	}
+	stage1RootFS := s.GetTreeStoreRootFS(stage1TreeStoreID)
+
+	// execute stage1's GC
+	if err := stage0.GC(p.path(), p.uuid, stage1RootFS, globalFlags.Debug); err != nil {
+		return fmt.Errorf("problem performing stage1 GC on %v, attempting brute-force GC: %v", p.uuid, err)
```

Comment:
You don't use this error string. Maybe print that instead of https://github.com/coreos/rkt/pull/1828/files#diff-204288dd8b5d5205881366c76eb4f97eR243?